netci.groovy

```diff
@@ -1432,74 +1501,123 @@ combinedScenarios.each { scenario ->
                                     buildNumber('${CORECLR_WINDOWS_BUILD}')
                                 }
                             }
+
+                            if (scenario == 'coverage') {
+                                // Move coreclr to clr directory
+                                shell("rm -rf .clr; mkdir .clr; mv * .clr; mv .git .clr; mv .clr clr")
+                                // Get corefx
+                                shell("git clone https://github.com/dotnet/corefx fx")
+                                shell("cd fx")
+                                // Build Linux corefx
+                                shell("./build.sh x64 release Linux skiptests")
+                                // Check contents of bin directory - this can be removed after we confirm everything is as expected
+                                shell("ls bin")
+                                shell("cd ${WORKSPACE}")
+
+                                // Run corefx tests
+                                shell("""./fx/run-test.sh \\
+                --coreclr-bins ${WORKSPACE}/clr/bin/Product/${osGroup}.${architecture}.${configuration} \\
+                --mscorlib-bins ${WORKSPACE}/clr/bin/Product/${osGroup}.${architecture}.${configuration} \\
+                --corefx-tests ${WORKSPACE}/fx/bin/tests/{osGroup}.AnyCPU.{configuration} \\
+                --corefx-native-bins ${WORKSPACE}/fx/bin/${osGroup}.${architecture}.${configuration}""")
+
+                                // Run coreclr GC tests w/ server GC enabled
+                                shell("""./clr/tests/runtest.sh \\
+                --testRootDir=\"\${WORKSPACE}/clr/bin/tests/Windows_NT.${architecture}.${configuration}/GC\" \\
```

Comment:
Ah, I see - looks like I misunderstood your last email. It sounds like the tests I want to run do get built, but are excluded on Linux. If they're known to be good on Linux, why don't we just take them out of TestsFailingOutsideWindows.txt? That way they'd run on Linux, but could still be excluded on Windows if need be.